examples/tests/valid/bring_cdktf.w

```diff
@@ -7,3 +8,11 @@ new aws.s3Bucket.S3Bucket(
     mfaDelete: true,
   },
 ) as "Bucket";
+
+class Foo {
+  init() {
+    // Test importing a non-standard Construct that doesn't have a `scope`/`id` property pair as its first two arguments
+    // Some of cdktf's backend constructs are like this.
+    new cdktf.S3Backend(this, cdktf.S3BackendConfig {bucket: "foo", key: "bar"});
+  }
+}
```

Comment:
Is it possible to override the `scope` and `id` of a preflight class in its `init()` function?
If the answer is yes, can you please add a test for that?

Reply:
It's not possible in Wing. Unconventional classes like `cdktf.S3Backend` do this, but I don't think we want to support it in wing.

Reply:
Ok, thanks. Asked because the spec mentioned it is supported, but I figured out it wasn't. I removed this section from the spec